src/coreclr/gc/gcpriv.h

```diff
@@ -419,8 +419,10 @@ enum gc_oh_num
     soh = 0,
     loh = 1,
     poh = 2,
-    none = 3,
-    total_oh_count = 4
+    free = 3,
+    none = 4,
+    total_oh_count = 3,
+    total_bucket_count = 5
```

Comment:
let's not use seemingly random integer values here. could replace by for example (poh + 1) for total_oh_count